btc-address-generation/src/main/kotlin/generation/btc/trigger/AddressGenerationExecutor.kt

```diff
@@ -0,0 +1,68 @@
+package generation.btc.trigger
+
+import com.github.kittinunf.result.flatMap
+import com.github.kittinunf.result.map
+import generation.btc.config.BtcAddressGenerationConfig
+import model.IrohaCredential
+import mu.KLogging
+import provider.TriggerProvider
+import provider.btc.BtcSessionProvider
+import provider.btc.address.BtcAddressType
+import sidechain.iroha.IrohaInitialization
+import sidechain.iroha.consumer.IrohaNetwork
+import sidechain.iroha.consumer.IrohaNetworkImpl
+import sidechain.iroha.util.ModelUtil
+
+private val logger = KLogging().logger
+
+//TODO springify this file
```

Comment:
Add issue in jira

Reply:
[Done](https://soramitsu.atlassian.net/browse/D3-622)